### PR TITLE
[13.0] Add shopinvader_notification_default 

### PR DESCRIPTION
**Pull request by simahawk**
FWD port of https://github.com/shopinvader/odoo-shopinvader/pull/525

Depends on #542

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -8,3 +8,6 @@ server-tools
 sale-workflow
 product-attribute
 odoo-misc https://github.com/shopinvader/odoo-misc 13.0
+social
+storage
+rest-framework
```
Comment (lmignon) [inline oca_dependencies.txt]: @simahawk storage and rest-framework are already declared

File: shopinvader/models/shopinvader_notification.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ class ShopinvaderNotification(models.Model):
     _description = "Shopinvader Notification"
 
     backend_id = fields.Many2one(
-        "shopinvader.backend", "Backend", required=True
+        "shopinvader.backend", "Backend", required=False
```
Comment (lmignon) [inline shopinvader/models/shopinvader_notification.py:15]: @simahawk Why do you make the backend optional?

**Review by lmignon — CHANGES_REQUESTED** [review]
image

File: shopinvader/models/shopinvader_notification.py (hunk 2 above)
Comment (simahawk) [reply shopinvader/models/shopinvader_notification.py:15]: because otherwise you cannot have predefined templates.
The fields is required in the view.

**Comment by simahawk** [comment]
@lmignon @Cedric-Pigeon would you mind to provide a bunch of example templates from your real projects? It would help speed up this :wink: :pray:

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by Cedric-Pigeon — APPROVED** [review]
Code review only

**Comment by Cedric-Pigeon** [comment]
> @lmignon @Cedric-Pigeon would you mind to provide a bunch of example templates from your real projects? It would help speed up this

Not sure it will help as their designs are highly plugged with our customer graphical chart and not really re-usable.

**Comment by simahawk** [comment]
@Cedric-Pigeon is mostly about the content, not the style.
If it's ok for you: send me your templates via email (or what you prefer) and I'll clean them up w/ very basic styling. Works for you?

**Pull request merged**